html2asketch/helpers/visibility.js

```diff
@@ -18,7 +18,10 @@ export function isNodeVisible(node, {width, height} = node.getBoundingClientRect
 } = getComputedStyle(node)) {
   // skip node when display is set to none for itself or an ancestor
   // helps us catch things such as <noscript>
-  if (node.tagName !== 'BODY' && node.offsetParent === null && position !== 'fixed') {
+  if (node.tagName !== 'BODY' &&
+      node.offsetParent === null &&
+      position !== 'fixed' &&
+      node.tagName.toLowerCase() !== 'slot') {
```

Comment:
can you please leave a comment in code about this special case?

Reply:
Sure thing.